connector/connect/server/src/main/scala/org/apache/spark/sql/connect/execution/SparkConnectPlanExecution.scala

```diff
@@ -163,7 +165,6 @@ private[execution] class SparkConnectPlanExecution(executeHolder: ExecuteHolder)
               // Collect errors and propagate them to the main thread.
               .andThen {
                 case Success(_) =>
-                  executePlan.eventsManager.postFinished()
```

Comment:
Is the empty case Success still needed to avoid an unexhaustive match, or rather is andThen taking a PartialFunction and just skipping case Success is fine? If it is not needed I'd remove; if it's needed, I'd add a comment like `// do nothing` instead of leaving it dangling like that.
(should be easy to test - if you remove it, and it expected it to be there, it should crash and burn everywhere, as this is the common code path; or it will work and it means it's fine)